.coveragerc

```diff
@@ -0,0 +1,11 @@
+[run]
+omit =
+    *tests/*
+    *examples/*
+    *benchmarks/*
+
+[report]
+omit =
+    *tests/*
+    *examples/*
+    *benchmarks/*
```

Comment:
i think this is unintentionally included, maybe from https://github.com/facebookresearch/torchsnapshot/pull/18 ? if so could please remove it from this PR

Reply:
Resolved.